TobiiTest/Preference.xaml

```diff
@@ -32,15 +32,28 @@
             </Separator.RenderTransform>
         </Separator>
 
-        <Label Name ="one" Content="Translator" HorizontalAlignment="Left" Margin="42,116,0,0" VerticalAlignment="Top" RenderTransformOrigin="-1.03,0.534" Height="25" HorizontalContentAlignment="Right" VerticalContentAlignment="Center"/>
-        <ComboBox x:Name="translatorCB"  HorizontalAlignment="Left" SelectedIndex="0" Margin="133,116,0,0" VerticalAlignment="Top" Width="135" Height="25" FontSize="10">
+        <Label Name ="one" Content="Translator" HorizontalAlignment="Left" Margin="80,116,0,0" VerticalAlignment="Top" RenderTransformOrigin="-1.03,0.534" Height="25" HorizontalContentAlignment="Right" VerticalContentAlignment="Center"/>
+        <ComboBox x:Name="translatorCB"  HorizontalAlignment="Left" SelectedIndex="0" Margin="194,116,0,0" VerticalAlignment="Top" Width="162" Height="25" FontSize="10">
             <ComboBox.Background>
                 <LinearGradientBrush EndPoint="0,1" StartPoint="0,0">
                     <GradientStop Color="#FFF0F0F0" Offset="0"/>
                     <GradientStop Color="#FFFBF8F8" Offset="1"/>
                 </LinearGradientBrush>
             </ComboBox.Background>
         </ComboBox>
+        
+        <Label Content="Set key" HorizontalAlignment="Left" Margin="97,150,0,0" VerticalAlignment="Top"/>
+        <ComboBox x:Name="setKey" HorizontalAlignment="Left" SelectedIndex="0" Margin="194,150,0,0" VerticalAlignment="Top"  Width="162" Height="25" FontSize="10">
+            <ComboBox.Background>
+                <LinearGradientBrush EndPoint="0,1" StartPoint="0,0">
+                    <GradientStop Color="#FFF0F0F0" Offset="0"/>
+                    <GradientStop Color="#FFFBF8F8" Offset="1"/>
+                </LinearGradientBrush>
+            </ComboBox.Background>
+        </ComboBox>
+
+        <Label Name="two" Content="Magnification factor" HorizontalAlignment="Left" Margin="25,186,0,0" VerticalAlignment="Top"/>
+        <TextBox x:Name="magnify" HorizontalAlignment="Left" Margin="194,186,0,0" TextWrapping="Wrap" Text="" VerticalAlignment="Top" Width="162" Height="25" FontSize="10"/>

```

Comment:
Can you add `NumberValidationTextBox` handler to this like in other two textboxes (Lines 12 and 13)? This is to ensure, that only numbers are able to be typed in these textboxes. I didn't check it myself though.
```suggestion
        <TextBox x:Name="magnify" HorizontalAlignment="Left" Margin="194,186,0,0" TextWrapping="Wrap" Text="" VerticalAlignment="Top" Width="162" Height="25" FontSize="10"  PreviewTextInput="NumberValidationTextBox"/>
```

Reply:
OK.